test/controllers/OtherReasonControllerSpec.scala

```diff
@@ -46,12 +48,15 @@ class OtherReasonControllerSpec extends SpecBase {
       any(), any())
     ).thenReturn(authResult)
 
+    when(mockUploadJourneyRepository.getUploadsForJourney(any())).thenReturn(Future.successful(None))
```

Comment:
Could we check the attribute in both the view and controller? For example, add a test here for when the repository returns some data?

Reply:
Added